test/src/main/scala/scalarules/test/scala_import/ScalaImportExposesJarsTest.scala

```diff
@@ -20,6 +20,11 @@ class ScalaImportExposesJarsTest extends SpecificationWithJUnit {
       success
     }
 
+    "stamp jars with a target label" in {
+      findManifest[Cache[String, String]] must haveTargetLabel
+      findManifest[ArrayUtils] must haveTargetLabel
+    }.pendingUntilFixed("runtime jars are not stamped")
```

Comment:
Will this test be ever fixed? Maybe just remove it or change assertion and test name to state that runtime jars are not stamped?

Reply:
Ideally we want stamps in runtime jars too. Currently this test serves as a gatekeeper if compile deps leak by accident to runtime. If you have better ideas how to express this situation, I would be happy to change this test. Please note that test like "jars must not be stamped" loses meaning of compile deps vs runtime deps information. If `pendingUntilFixed` fails, it would indicate potential mistake in the code.

Reply:
> Ideally we want stamps in runtime jars too. 

Why? There is no need for this and not possible for signed jars

> Currently this test serves as a gatekeeper if compile deps leak by accident to runtime. 

As I understand this test checks just that https://github.com/bazelbuild/rules_scala/pull/1214/files#diff-815ff37ec22a035579a0711eb9d9ff114a7cede2c7e4ffa73af10ae158994035R28-R43

> Please note that test like "jars must not be stamped" loses meaning of compile deps vs runtime deps information. 

Not sure what do you mean. How this meaning is preserved now? To me `pendingUntilFixed` looks like a fancy way to invert assertion.

> If pendingUntilFixed fails, it would indicate potential mistake in the code.

For me it reads like a TODO which will never be fixed. If mistake is made then test should fail assertion and not pass it.


In any case this is really a minor thing and should not block the merge. It's just hard for me to understand the intent of this test. Feel free to merge.

Reply:
Removed tests